caches/path-mapped/src/main/java/org/commonjava/maven/galley/cache/pathmapped/model/ReverseMap.java

```diff
@@ -15,13 +17,13 @@
     private ReverseKey reverseKey;
 
     @Column
-    private String paths;
+    private HashSet<String> paths;
```

Comment:
I found this annotation:  @ElementCollection 
( When the Map value is an embeddable class or basic type, use the @ElementCollection annotation. )
Do we need it ?

Reply:
Using this one will create another table and JPA joins these two tables when querying/updating/etc. Cassandra does not support this joining so we have to avoid it.

Reply:
ACK, i see.